packages/medusa/src/services/cart.js

```diff
@@ -1,5 +1,5 @@
 import _ from "lodash"
-import { Validator, MedusaError } from "medusa-core-utils"
+import { MedusaError, Validator } from "medusa-core-utils"
```

Comment:
Why are we using a validator in the service layer, should be in the api, remember for refactor when removing the entity manager from the api 😄